app.js

```diff
@@ -129,29 +188,59 @@ function header(message) {
  * @param {puppeteer.Browser} browser
  * @returns {Promise<boolean>} Are there any drops?
  */
-async function hasValorantDrop(browser) {
+async function hasDrops(browser) {
   header("Dropcheck");
   debug("Opening inventory page ...");
   const page = await browser.newPage();
   await page.goto(`${BASE_URL}/inventory`, { waitUntil: "networkidle2" });
   debug("Querying for drops ...");
-  const noDropsElement = await page.$(NO_DROPS_QUERY);
-  const noDrops = noDropsElement === null;
+  
+  /* Updated by AlexSimpler */
+  let noDrops = await query(page, NO_DROPS_QUERY);
+  //if there are noDrops then length > 0
+  if (noDrops.length) {
+    info("Haven't received a drop yet");
+  }
+  else {
+    //wait for some time before querying to avoid some element not found errors
+    await idle(1000);
+    
+    let count = 0;
+    let drop = await query(page, DROP_INVENTORY_LIST);
+    count = (await query(page, DROP_INVENTORY_LIST + ">" + DROP_ITEM)).length;
+    let received = false;
+
+    if (count) {
+      //just itterate through each notification and break if one drop has the name of the game name uppercase
+      for (let i = 0; i < count; i++) {
+        let game = (await query(page, `${DROP_INVENTORY_LIST + ">" + DROP_ITEM}:nth-child(${i + 1}) ${DROP_INVENTORY_NAME}`))
+          .text().toUpperCase();
+        if (game == CONFIG_FILE.game.toUpperCase()) {
+          success = true;
+          break;
+        }
+      }
+      await idle(1500);
+      spinner.stop(1);
+      if (received) {
+        success(`🎉 Congrats you got ${(upperFirst(CONFIG_FILE.game))}!`);
```

Comment:
We can skip the emoji, it doesn't show up right on windows

Reply:
Ok